metadata/service/server/cluster.yml

```diff
@@ -12,3 +12,4 @@ parameters:
       cluster:
         name: elasticsearch
         multicast: true
+      enable_script: true
```

Comment:
you shouldn't change the previous setting. Please modify the user model instead.